mesh_layers/mesh_layers.xml

```diff
@@ -1,4 +1,11 @@
 <library path="mesh_layers">
+    <class name="mesh_layers/BorderLayer"
+            type="mesh_layers::BorderLayer"
+            base_class_type="mesh_map::AbstractLayer">
+        <description>
+            A border layer, setting the border vertices of a mesh to high costs (dangerous).
```

Comment:
First intuition when reading the description was that using this layer is dangerous, rather than that high costs indicate danger. But maybe that's just me :D